pycbc/psd/variation.py

```diff
@@ -59,24 +67,22 @@ def calc_psd_variation(strain, psd_short_segment, psd_long_segment,
         # Calculate PSD for long segment and separate the long segment in to
         # overlapping shorter segments
         if tlong + psd_long_segment <= end_time:
-            psd_long = strain.time_slice(tlong,
-                          tlong + psd_long_segment).psd(overlap)
+            psd_long = pycbc.psd.welch(strain.time_slice(tlong, tlong + psd_long_segment), seg_len=int(short_psd_duration * strain.sample_rate), seg_stride=int(short_psd_stride * strain.sample_rate), avg_method=psd_avg_method)
```

Comment:
There are now some very long lines in this code, can they be reformatted to be max 79 characters (except for rare exceptions) - see PEP8 ?

Reply:
updated - i agree reads much better. Just let me know if I didn't follow pep8 well enough